src/styles/globals.css

```diff
@@ -2,3 +2,127 @@
 @tailwind components;
 @tailwind utilities;
 
+.lds-dual-ring {
```

Comment:
I see alot of conflicts with #16

Reply:
I imported the styles and it seemed fine

Reply:
As in #16 is using sass, not too sure if there is a clean way to resolve this, prob port this over to a sep sass file

Reply:
I moved all the styling into the sass file in the new commit